composer.json

```diff
@@ -21,7 +21,8 @@
     "require": {
         "victoire/victoire": "~2.3 | ~3.0",
         "troopers/alertify-bundle": "~3.0",
-        "egeloen/ckeditor-bundle": "~4.0"
+        "egeloen/ckeditor-bundle": "~4.0",
+        "google/recaptcha": "~1.1"
```

Comment:
I think that the recaptcha should not be installed by default. It may be moved in the `suggest` part (but only if it doesn't require too much work).